app/templates/admin/add_tag.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'layouts/base.html' %}
+{% import 'macros/form_macros.html' as f %}
+
+{% block content %}
+<div class="row">
+    <div class="col back-button">

```

Comment:
Did you change something with the margins of this page between the last two PR's? This is what it looks like for me: 
<img width="1140" alt="screen shot 2018-04-12 at 2 39 14 pm" src="https://user-images.githubusercontent.com/9835059/38697188-49a09fbc-3e5f-11e8-8e90-b26d9af98ec3.png">

Reply:
This is  as a result of bootstrap

Reply:
All that needs to be done is to wrap this entire  thing in a 
<div class="container">